src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -229,12 +229,49 @@ export class AppRoot extends mixinBehaviors
       #appDrawer a:focus {
         outline: none;
       }
+      #links-footer {
+        margin-top: 36px;
+      }
+      #language-icon {
+        padding-top: 10px;
+      }
+      #language-dropdown {
+        padding-left: 22px;
+        --paper-menu-button: {
+          padding: 0;
+        };
+        --paper-input-container: {
+          width: 156px;
+        };
+        --paper-input-container-input: {
+          color: var(--medium-gray);
+          font-size: 14px;
+        };
+      }
       .legal-links {
-        margin: 36px -6px 0;
+        margin: 0 -6px 0;
       }
       .legal-links > * {
         margin: 0 6px;
       }
+      #language-row {
+        display: flex;
+        align-items: center;
+      }
+      .language-item {
+        display: flex;
+        cursor: pointer;
+        font-size: 16px;
+        padding-left: 24px;
+        --paper-item-selected: {
+          color: var(--primary-green);
+          font-weight: normal;
+        }
+      }
+      .language-name {
+        text-align: left;
```

Comment:
Is this needed? Won't the dir property cause this to be set appropriately?

Reply:
Good catch, it's not! Removed.